1_🔍_Search.py

```diff
@@ -234,18 +230,19 @@ def get_cached_response(_query_engine, _slack_client, query):
 
 st.sidebar.markdown(SIDEBAR_DESC)
 
-if 'survey_complete' not in st.session_state:
-    st.session_state.survey_complete = True
-if 'query' not in st.session_state:
-    st.session_state.query = ""
-if 'response' not in st.session_state:
-    st.session_state.response = None
-if 'response_error' not in st.session_state:
-    st.session_state.response_error = False
-if 'db_id' not in st.session_state:
-    st.session_state.db_id = None
-if 'last_processed_query' not in st.session_state:
-    st.session_state.last_processed_query = ""
+# Initialize session state for search functionality
+default_states = {
+    'survey_complete': True,
+    'query': "",
+    'response': None,
+    'response_error': False,
+    'db_id': None,
+    'last_processed_query': ""
+}
+
+for key, default_value in default_states.items():
+    if key not in st.session_state:
```

Comment:
There is also a default state initialization in `state.py`. Could these defaults be moved over there?